app/views/dashboard/_latest-jobs.html.erb

```diff
@@ -0,0 +1,17 @@
+<% stylesheet 'foreman_remote_execution/foreman_remote_execution' %>
+
+<h4 class="header"><%= link_to _("Latest Jobs"), job_invocations_path(:order=>'started_at DESC')%></h4>
+<table class="<%= table_css_classes('table-fixed') %>">
+  <tr>
+    <th class="col-md-5"><%= _("Name") %></th>
+    <th class="col-md-2"><%= _("State") %></th>
+    <th class="col-md-3"><%= _("Started") %></th>
+  </tr>
+  <% JobInvocation.new.latest_jobs.each do |invocation| %>
```

Comment:
we shouldn't need to build the instance to load other job invocation instances, either make the `latest_jobs` a class method or convert it to the named scope